client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -81,9 +81,17 @@
 
 .madlib__input:empty {
 	border-bottom: 2px solid transparent;
-	position: relative;
+	display: inline-block;
```

Comment:
I don't think it needs to have a different `display` value since it's absolutely positioned. In this [draft PR](https://github.com/Automattic/wp-calypso/pull/42146/files) of mine it was working fine being inline.